identity-admin/webapp/components/search/StatusDropdown.tsx

```diff
@@ -0,0 +1,27 @@
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import { buildSearchUrl } from '../../utils/search-util';
+
+const StatusDropdown = (): JSX.Element => {
+  const router = useRouter();
+  const { status, name, email } = router.query;
+  const [statusValue] = useState<string>(
+    typeof status === 'string' ? status || '' : ''
+  );
+
+  const onChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    window.location.href = buildSearchUrl('1', event.target.value, name, email);
+  };
+
+  return (
+    <select onChange={onChange} value={statusValue}>
+      <option value="">Select status</option>
```

Comment:
We can prevent this option from appearing in the menu or being selected:

```suggestion
      <option hidden>Select status</option>
```